### ISSUE TITLE
add fisheyegl module

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#27]
Hi, @ccpandhare - whats the latest? 

I wanted to add this as a feature request down the line -- it's a good one to try to develop a test for:

https://github.com/jywarren/fisheyegl

could be a great module. I want to suggest a test that takes a checkerboard image and corrects fisheye, then diff-compares it with a correct checkerboard.

**Comment by ccpandhare** [comment]
Currently, I am designing tests for the core functionality.

Sure, I will add this module in.

**Comment by ccpandhare** [comment]
I had a look at the code. The module works on canvasses so I'll have to figure out how to make things work for Node.js and that may take some time...

**Comment by jywarren** [comment]
Low priority!!

**Comment by ccpandhare** [comment]
Ohh okay. Also, I am working on a unified input parser.

**Comment by jywarren** [comment]
I think it should be OK to have some modules which are marked as working only in the browser. In theory, we could shim this so it'd boot up a phantomjs instance, too, but we can think about that later. 

I mentioned some interest from NASA folks and I think this may be a good project that they could use. I'd like to get a demo running with a `crop` module and then pass the output to this `lensDistortion` module, and have a web-based demo of it -- what do you think? It'd be a great proof of your work to date, even if it doesn't yet work in Node.

**Comment by ccpandhare** [comment]
Okay! Wow!
Sounds great!!

**Comment by ccpandhare** [comment]
I think I can work on that demo now.

**Comment by jywarren** [comment]
Oh great!

**Comment by ccpandhare** [comment]
I have ported the Latest FisheyeGL version from `jywarren/fisheyegl` to ImageSequencer.
I am adding it to documentation and will create a Pull Request soon!

**Comment by jywarren** [comment]
Will we need to adjust the module to point at the fisheyegl mom package?

**Comment by ccpandhare** [comment]
Well once the README.md of `jywarren/fisheyegl` is complete and is published to npm, we can just require in `fisheyegl` from npm and use it directly. I shall complete that immediately.

**Comment by jywarren** [comment]
fisheyegl `v0.1.0` now published :-)

**Comment by ccpandhare** [comment]
Okay thanks!

**Comment by ccpandhare** [comment]
I think it hasn't updated yet.
[npm/fisheyegl](https://www.npmjs.com/package/fisheyegl) still shows the old README.md and 
```
npm i fisheyegl
```
is still serving the old files

**Comment by jywarren** [comment]
hmm, i didn't include the README but the files should be new. i'll issue
v0.1.1 to be sure.

On Wed, Jul 26, 2017 at 10:18 PM, Chinmay Pandhare <notifications@github.com

**Comment by ccpandhare** [comment]
Now it works fine, thanks!

**Comment by jywarren** [comment]
great.
